src/styles/_themes.scss

```diff
@@ -0,0 +1,71 @@
+@import 'styles/sfds/_colors.scss';
+
+// This file contains css properties use throughout the
+// code base for each corresponding theme.
+// Loaded in next.config.js under sassOptions
+
+/************** Light Theme (default) **************/
+
+:root {
```

Comment:
Question: For browsers that don't support variables will they fallback to the default theme? Or will it just not work?

Reply:
Because the default theme still uses css variables, my understanding is it wouldn't work. But I don't know what that looks like visually. For the record, all browsers we support also support css vars.